tests/spec_tests/query_tests/information_schema_lowercase.yml

```diff
@@ -1,20 +1,6 @@
-catalog_data:
-  information_schema:
-    mytbl:
-      - {'_id': 0, 'a': 1}
-
-catalog_schema:
-  {
-    'information_schema': {
-      'mytbl': {
-        'bsonType': 'object',
-        'additionalProperties': true,
-      }
-    }
-  }
-
 tests:
   - description: a user db called information_schema should require no special handling
+    skip_reason: "SQL-xxx: Should this actually be part of the spec?"
```

Comment:
I added this skip_reason because this test in particular causes a bit of a dilemma... MongoSQL is case sensitive, but _MongoDB_ is not. As in, we can't create 2 databases called `INFORMATION_SCHEMA` and `information_schema` in mongod. The old way of running these was fine since we dropped all data before running a test. But now that we load everything upfront, we get an error when we attempt to create `information_schema` after creating `INFORMATION_SCHEMA`.

I argue that this test is not even worth keeping. The CAPS-only version is sufficient to demonstrate the "information schema" db does not exist in MongoSQL. What do you all think?

Reply:
I think that's a good reasoning.  Seemed like it's more of a difference in how the tests were run itself so I'm okay with removing it.